File: Java/FloydsTriangle.java
// Github Username = Tikshita-Singh
//JAVA program to print Floyds Triangle

import java.util.*;

public class FloydsTriangle {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter the number of rows");
		int n = sc.nextInt();	
		int t = 1;
		for(int i = 1;i <= n;i++) {
			for(int j = 1;j <= i;j++) {
				System.out.print(t + " ");
				t++;
			}
			System.out.println();
		}
	}
}

File: CPP/Floyds_Triangle.cpp
// GitHub Username : nitishdeshpande07
// Email : nitishdeshpande07@gmail.com

// This is a program to print a floyds triangle of any given required size 

#include<iostream>

using namespace std ;

int main()
{
    int n ;
    int m=1 ;
    cout<<"enter the number of rows upto which the floyds traingle is intended to be printed "<<endl;
    cin>>n;

for(int j=2;j<=n;j++)
{ 
    cout<<"\n";
    for(int i=1;i<j;i++)
    {
        cout<<m;
        m++;
        


    }
}

return 0 ;

}
